### DISCUSSION TITLE
Failed fetching content exclusion policies

### DISCUSSION THREAD

**Discussion by BaoTrung37** [community/community#141200] [Other Feature Feedback, Questions, & Ideas]
```
2024-10-12 01:35:59.826 [error] [contentExclusion] Failed fetching content exclusion policies {
  params: {
    repos: 'https://github.com/...git',
    scope: 'repo'
  },
  data: { message: 'Server Error' }
}```
I can't access to copilot. pls help me

**Comment by dpasukhi** [comment]
The same issue:
```
2024-10-11 19:28:06.033 [error] [contentExclusion] Failed fetching content exclusion policies {
  params: { repos: 'https://github.com/dpasukhi/occt.git', scope: 'repo' },
  data: { message: 'Server Error' }
}
2024-10-11 19:28:06.037 [error] [contentExclusion] FetchResponseError: HTTP 502 
    at /home/coder/.vscode-server-insiders/extensions/github.copilot-1.238.0/lib/src/contentExclusion/contentExclusions.ts:243:19
    at processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 'HTTP502'
} Error evaluating policy for <file:///home/coder/work/OCCT/src/DE/DE_Wrapper.cxx>
```

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @BaoTrung37 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category and including as many relevant details as possible. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by joemar25** [comment]
It is Probably due to a temporary server issue that’s affecting content exclusion policies. You could try closing and reopening your code editor, signing out of Copilot and signing back in, or checking for any updates to the Copilot extension. If that doesn’t fix it, it might just be a temporary glitch on GitHub’s side, and waiting a bit could do the trick. But if the issue keeps happening, it’s a good idea to reach out to GitHub support or check their status page to see if there are any known problems.
